packages/flutter/lib/src/widgets/app.dart

```diff
@@ -185,13 +189,13 @@ class WidgetsApp extends StatefulWidget {
 
 class _WidgetsAppState extends State<WidgetsApp> implements WidgetsBindingObserver {
   GlobalObjectKey<NavigatorState> _navigator;
-  LocaleQueryData _localeData;
+  Locale _locale;
```

Comment:
did you not like this suggestion? (that's fine, just curious if you missed it or dismissed it)